regimes/se/scenarios.go

```diff
@@ -0,0 +1,38 @@
+package se
+
+import (
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/org"
+	"github.com/invopop/gobl/tax"
+)
+
+var invoiceScenarios = &tax.ScenarioSet{
+	Schema: bill.ShortSchemaInvoice,
+	List: []*tax.Scenario{
+		// Reverse Charges
+		// Reverse charges can only happen if the customer is NOT registered
+		// in Sweden (doesn't have a Swedish VAT ID, usually a foreign company),
+		// or if the supplier is in real estate, not registered in Sweden,
+		// and the customer has a Swedish VAT ID.
+		// It can never happen for national operations (i.e. both parties are registered in Sweden,
+		// whether B2B or B2C).
+		{
+			Tags: []cbc.Key{tax.TagReverseCharge},
+			Note: &tax.ScenarioNote{
+				Key:  org.NoteKeyLegal,
+				Src:  tax.TagReverseCharge,
+				Text: "Reverse charge: Customer to account for VAT to the relevant tax authority.",
+			},
+		},
+		// Simplified Tax Invoice
+		{
+			Tags: []cbc.Key{tax.TagSimplified},
+			Note: &tax.ScenarioNote{
+				Key:  org.NoteKeyLegal,
+				Src:  tax.TagSimplified,
+				Text: "Simplified Tax Invoice",
+			},
+		},
```

Comment:
This probably isn't required, as simplified is often implied when customer not present..

Reply:
Yeah in that case I'll simply use the generic `InvoiceScenarios` as you pointed out. I was in between using it directly and expanding it with the `TagSimplified` and this. Followed what other regimes did for consistency.